service/src/main/resources/api/openapi.yml

```diff
@@ -523,6 +549,62 @@ components:
           items:
             $ref: '#/components/schemas/FailureMessages'
 
+    SubmitWorkflowRequestBody:
+      type: object
+      properties:
+        workflowGcsUri:
+          description: GCS uri which points to the workflow. Either workflow GCS uri or workflow url is required. If both are specified, then workflow URL takes precedence.
+          type: string
+        workflowUrl:
+          description: URL which points to the workflow. Either workflow GCS uri or workflow url is required. If both are specified, then workflow URL takes precedence.
+          type: string
+        workflowOnHold:
+          description: Put workflow on hold upon submission. By default, it is taken as false.
+          type: boolean
+          default: false
+        workflowInputs:
+          description: JSON or YAML file containing the inputs as an object. For WDL workflows a skeleton file can be generated from WOMtool using the "inputs" subcommand.
+          type: string
+        workflowOptions:
+          description: JSON file containing configuration options for the execution of this workflow. jes_gcs_root is required; the rest are optional.
+          type: object
+          additionalProperties: true
+          properties:
+            jes_gcs_root:
+              type: string
+            delete_intermediate_output_files:
+              type: string
+            memory_retry_multiplier:
+              type: string
+            write_to_cache:
+              type: string
+            read_from_cache:
+              type: string
+            final_workflow_log_dir:
+              type: string
+            final_call_logs_dir:
+              type: string
+          required:
+            - jes_gcs_root
+        workflowType:
+          description: The workflow language for the file you submitted. Cromwell currently supports WDL.
+          type: string
+          enum: [ WDL ]
+        workflowTypeVersion:
+          description: The specification version for the workflow language being used. For WDL, Cromwell currently supports draft-2 and 1.0.
+          type: string
+          enum: [ "1.0", "draft-2"]
```

Comment:
there's the weirdest enum form I've seen...

Reply:
it is pretty weird isnt it